test/remote/gateways/remote_vpos_test.rb

```diff
@@ -4,7 +4,8 @@ class RemoteVposTest < Test::Unit::TestCase
   def setup
     @gateway = VposGateway.new(fixtures(:vpos))
 
-    @amount = 100000
+    # some test fails due duplicated transactions
+    @amount = rand(100..100000)
```

Comment:
Additionally I made a small change to the amount to avoid fails due duplicated transactions